.github/DEVELOPER.md

```diff
@@ -27,12 +27,23 @@ create
 Using the `Makefile` you can use `make run` to run instance-manager locally on your machine, and it will try to reconcile InstanceGroups in the cluster.
 Make sure the context you are in is correct and that you scaled down the actual instance-manager controller pod.
 
+1. You need to create 2 files for controller config and cfn and copy the data section from #config/crd/bases/instance-manager-configmap.yaml respectively
+2. Make sure you have AWS Credentials and region exported so that your local instance-manager controller can execute a cloud formation template
+3. Update the run section with the file locations (for example like below)
+```bash
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet
+	go run ./main.go --controller-config /Users/nmogulla/controller.conf --controller-template /Users/nmogulla/cloudformation.template
```

Comment:
Thanks for adding this, 3. is actually not needed, maybe better to ask people to place `controller.conf` and `cloudformation.template` under `/etc/config` locally which is the default location? that way no need to modify the Makefile.

Reply:
At least in my work laptop, it is not allowing me to add a file in /etc/config even with root and probably it is going to be an issue if somebody is working on windows OS(may be??)

Reply:
You can add it after you `sudo su`
Let's not worry about windows users for now :))))

Reply:
Done! Removed 3rd point